src/App.tsx

```diff
@@ -21,55 +21,125 @@ enum SortType {
   LENGTH,
 }
 
-// Use this function in the render method
+// DON'T save goods to the state
+type State = {
+  isStarted: boolean,
+  isReversed: boolean,
+  sortType: SortType,
+};
+
 function getReorderedGoods(
   goods: string[],
   sortType: SortType,
   isReversed: boolean,
 ) {
-  // Not to mutate the original array
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // ...
+  if (sortType === SortType.ALPABET) {
+    visibleGoods.sort((a, b) => a.localeCompare(b));
+  } else if (sortType === SortType.LENGTH) {
+    visibleGoods.sort((a, b) => a.length - b.length);
+  }
```

Comment:
could have used switch statement

Reply:
It will be better to use switch case construction.